recipes/assimp/5.x/test_package/conanfile.py

```diff
@@ -29,3 +29,8 @@ def test(self):
 
             bin_c_path = os.path.join(self.cpp.build.bindirs[0], "test_package_c")
             self.run(f"{bin_c_path} {obj_path}", env="conanrun")
+
+            # Check that a GLTF2 file with draco compression can be loaded without error
```

Comment:
Hi @bacusters  - we no longer allow checking binary files into the repository. Please note that the intended purpose of the test package is to check that `find_package` works in CMake, that the compiler can resolve includes from the package, and that the linker can correctly resolve the library and symbols from the binary files , and *not* a functionality check. Apologies as some legacy test packages still don't reflect this.